### ISSUE TITLE
 Provide Finance.Iban() with argument to generate IBANs from specific countries

### ISSUE THREAD

**Issue by Tsingis** [bchavez/Bogus#229]
I don't know how easy this would be to implement but an array of ISO 3166 country codes feels like clever choice. On default all it would work like it currently does.

**Comment by bchavez** [comment]
Hi Jesse,

Thanks for creating the GH issue. I'll take a look at this after I'm done working on #225 and #230. 

A `string[]` array seems okay. However, I think also, a simple `string` parameter could work too.

My only immediate thought with using a `string[]` parameter is that it implies some distribution among items in the `string[]`.  IE: an implicit weight distribution among iso3166 codes would be `.5`.

When the parameter is just a `string` focusing on the most primitive denormalized type, composing a custom distribution with `.Iban(string countryCode)` just boils down to creating a few operations before calling the main `.Iban(string)` method. For example,

```csharp
var faker = new Faker();
var codes   = new []{"US", "UK", "CA", "MX"};
var weights = new []{ .2f,  .5f,  .2f,  .1f};

var code = faker.Random.WeightedRandom(codes, weights);
faker.Finance.Iban(code);
```

Of course, we could have both parameters, but this is just my initial thought.

Thanks again for creating the issue. I'll try to get to it soon.
Brian

**Comment by Tsingis** [comment]
Good thinking. I assumed even distribution but this would be much better.

**Comment by bchavez** [comment]
Hi Jesse,

`.Iban(countryCode)` should now be available in [Bogus v28.0.1](https://www.nuget.org/packages/Bogus/28.0.1).

Hope it works for you. Let me know how it goes.

Thanks,
Brian
